### PR TITLE
FUI-2126: entity-management new params

### PR DESCRIPTION
**Pull request by ArturKrasinski**
A minor update to the entity-management documentation (new parameters related to CRUD buttons have been added).

**Please do not merge until this PR: [LINK](https://github.com/genesislcap/foundation-ui/pull/1577) has been merged.**

----

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?
No - current only

Have you checked all new or changed links?
Yes, there are no changes.

Is there anything else you would like us to know?
No

For reference: 

  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 

**This week's exciting excerpts from the style guide**

- We write in UK English, with Oxford English Dictionary (OED) spellings, grammar and vocabulary.  

Use the present tense wherever possible. Only use another tense where it is strictly necessary.

- Present tense (preferred whenever possible): *The script creates a new folder.*
- Future tense (avoid whenever possible): *The script will create a new folder.*

### CODE REVIEW

File: docs/04_web/05_micro-front-ends/03_foundation-entity-management/03_foundation-entity-management.mdx
Comment (wjhendry) [inline docs/04_web/05_micro-front-ends/03_foundation-entity-management/03_foundation-entity-management.mdx:1]: What are the permitted values for:
- crud-menu-position
- crud-menu-style
?

Comment (wjhendry) [reply docs/04_web/05_micro-front-ends/03_foundation-entity-management/03_foundation-entity-management.mdx:1]: dziekuje!

Comment (wjhendry) [reply docs/04_web/05_micro-front-ends/03_foundation-entity-management/03_foundation-entity-management.mdx:1]: None = 'none',
does that mean it won't be displayed?

Comment (wjhendry) [reply docs/04_web/05_micro-front-ends/03_foundation-entity-management/03_foundation-entity-management.mdx:1]: And style:
Default = 'default', what is the default?

Comment (ArturKrasinski) [reply docs/04_web/05_micro-front-ends/03_foundation-entity-management/03_foundation-entity-management.mdx:1]: > None = 'none', does that mean it won't be displayed?

Yes, exactly. Setting None = 'none' means it won’t be displayed.

> Default = 'default', what is the default?

The default behavior will display the buttons side by side, as it currently does (not as an action menu or any other list in the future).

**Review by wjhendry — APPROVED** [review]

**Pull request merged**